style.css

```diff
@@ -0,0 +1,61 @@
+*   {
+    text-align: center
+}
+header {
+    background: red;
+    padding: 1px;
```

Comment:
I see that you have some same styling like paddings and margins for multiple classes, best thing you can do to clean up the code is to add another class to the same div. You can add multiple classes to one id
Ex: `<div class="classOne classTwo">`